packages/search/src/features/search/utils.ts

```diff
@@ -13,272 +13,181 @@ import {
   CERTIFIED_STATUS,
   REGISTERED_STATUS
 } from '@search/elasticsearch/utils'
-import {
-  IAdvancedSearchParam,
-  IFilter,
-  INameCombination
-} from '@search/features/search/types'
-const SEARCHABLE_FIELDS = [
-  'childFirstNames',
-  'childFamilyName',
-  'childFirstNamesLocal',
-  'childFamilyNameLocal',
-  'deceasedFirstNames',
-  'deceasedFamilyName',
-  'deceasedFirstNamesLocal',
-  'deceasedFamilyNameLocal',
-  'trackingId',
-  'registrationNumber',
-  'contactNumber'
-]
-
-const allNameFields = [
-  'childFirstNames',
-  'childFamilyName',
-  'childFirstNamesLocal',
-  'childFamilyNameLocal',
-  'motherFirstNames',
-  'motherFamilyName',
-  'motherFirstNamesLocal',
-  'motherFamilyNameLocal',
-  'fatherFirstNames',
-  'fatherFamilyName',
-  'fatherFirstNamesLocal',
-  'fatherFamilyNameLocal',
-  'informantFirstNames',
-  'informantFamilyName',
-  'informantFirstNamesLocal',
-  'informantFamilyNameLocal',
-  'deceasedFirstNames',
-  'deceasedFamilyName',
-  'deceasedFirstNamesLocal',
-  'deceasedFamilyNameLocal',
-  'spouseFirstNames',
-  'spouseFamilyName',
-  'spouseFirstNamesLocal',
-  'spouseFamilyNameLocal'
-]
-
-const childFirstNameFields = ['childFirstNames', 'childFirstNamesLocal']
-
-const motherFirstNameFields = ['motherFirstNames', 'motherFirstNamesLocal']
-
-const fatherFirstNameFields = ['fatherFirstNames', 'fatherFirstNamesLocal']
-
-const childFamilyNameFields = ['childFamilyName', 'childFamilyNameLocal']
-
-const motherFamilyNameFields = ['motherFamilyName', 'motherFamilyNameLocal']
-
-const fatherFamilyNameFields = ['fatherFamilyName', 'fatherFamilyNameLocal']
-
-export const EMPTY_STRING = ''
-
-export function queryBuilder(
-  query: string,
-  trackingId: string,
-  nationalId: string,
-  contactNumber: string,
-  registrationNumber: string,
-  eventLocationId: string,
-  gender: string,
-  nameCombinations: INameCombination[],
```

Comment:
@euanmillar I'm just concerned if we need to keep the `nameCombinations` prop here. Although I don't think it's being used anywhere that's why Sadman removed it